src/main/java/org/cascadebot/cascadebot/utils/DiscordUtils.java

```diff
@@ -67,12 +68,15 @@ public static User getUser(Guild guild, String search, boolean retrieve) {
         Checks.notBlank(search, "user");
         List<User> users = FinderUtil.findUsers(search, guild.getJDA());
         if (users.size() > 1) {
-            /*context.replyDanger("There is more than one user that matches this criterion! Please enter the ID or the user's full name!");*/
```

Comment:
When I said uncomment this, I didn't mean delete it 😂

Reply:
We don't have context passed through so we can't reply like that
*binaryoverload said to comment it back in & say why on discord